MekHQ/src/mekhq/campaign/parts/TankLocation.java

```diff
@@ -207,7 +207,7 @@ public void updateConditionFromEntity(boolean checkForDestruction) {
             if(IArmorState.ARMOR_DESTROYED == unit.getEntity().getInternal(loc)) {
                 remove(false);
             } else {
-                damage = unit.getEntity().getOInternal(loc) - unit.getEntity().getInternal(loc);
+                damage = Math.max(unit.getEntity().getOInternal(loc) - unit.getEntity().getInternal(loc), 0);
```

Comment:
If getInternal(loc) is `IArmorState.ARMOR_DESTROYED`, `damage` will be original internal + 3.

Reply:
Agreed, but isn't that checked above?

Reply:
So it is. I was only looking at the line that was changed on not the surrounding context.

Reply:
I've updated it to cap damage at original internals.